app/components/Navbar.jsx

```diff
@@ -1,79 +1,103 @@
-            </Link>
-            <ul style={{color: `${textColor}`}} className='hidden sm:flex'>
-                <li className='p-4'>
-                    <Link href='/'>Home</Link>
-                </li>
-                <li className='p-4'>
-                    <Link href='/#gallery'>Gallery</Link> 
-                </li>
-                <li className='p-4'>
-                    <Link href='/work'>Work</Link>
-                </li>
-                <li className='p-4'>
-                    <Link href='/contact'>Contact</Link>
-                </li>
-            </ul>
+    <div
+      style={{ backgroundColor: `${color}` }}
+      className="fixed left-0 top-0 w-screen z-10 ease-in duration-300"
+    >
+      <div className="m-auto flex justify-between items-center p-4 text-white ">
+        <Link href="/">
+          <h1 style={{ color: `${textColor}` }} className="font-bold text-4xl">
+            Moments
+          </h1>
+        </Link>
+        <ul style={{ color: `${textColor}` }} className={`max-sm:hidden flex`}>
+          <li className="p-4">
+            <Link href="/">Home</Link>
+          </li>
+          <li className="p-4">
+            <Link href="/#gallery">Gallery</Link>
+          </li>
+          <li className="p-4">
+            <Link href="/work">Work</Link>
+          </li>
+          <li className="p-4">
+            <Link href="/contact">Contact</Link>
+          </li>
+        </ul>
 
-            {/* Mobile Button */}
-            <div className='block sm:hidden z-10'>
-               {nav ? <AiOutlineClose size={20} style={{color: `${textColor}`}} /> : <AiOutlineMenu size={20} style={{color: `${textColor}`}} />}
-            </div>
-            {/* Mobile Menu */}
-            <div className= {
-            nav ? 'sm:hidden absolute top-0 left-0 right-0 bottom-0 flex justify-center items-center w-full h-screen bg-black text-center ease-in duration-300' : 'sm:hidden absolute top-0 left-[-100%] right-0 bottom-0 flex justify-center items-center w-full h-screen bg-black text-center ease-in duration-300'}>
-                <ul>
-                    <li onClick={handleNav} className='p-4 text-4xl hover:text-gray-500'>
-                        <Link href='/'>Home</Link>
-                    </li>
-                    <li onClick={handleNav} className='p-4 text-4xl hover:text-gray-500'>
-                        <Link href='/#gallery'>Gallery</Link> 
-                    </li>  
-                    <li onClick={handleNav} className='p-4 text-4xl hover:text-gray-500'>
-                        <Link href='/work'>Work</Link>
-                    </li>
-                    <li onClick={handleNav} className='p-4 text-4xl hover:text-gray-500'>
-                        <Link href='/contact'>Contact</Link>
-                    </li>
-                </ul>
-            </div>
+        {/* Mobile Button */}
+        <div className="md:hidden block z-10" onClick={handleNav}>
```

Comment:
This is why your navbar didn't show up. you need `onClick` on the trigger button which is the hamburger menu